cli/BUILD.bazel

```diff
@@ -1,5 +1,5 @@
-load("@rules_python//python:defs.bzl", "py_binary")
 load("@pip//:requirements.bzl", "requirement")
+load("@rules_python//python:defs.bzl", "py_binary")
```

Comment:
I always have a bit of anguish around changes like this. It was semantically better before with the base language before the dependency management system, but it is syntactically more consistent now to have them sorted.